_posts/2023-10-26-install-vivaldi.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Installing Vivaldi
+description: Learn how to Install and set up Vivaldi in Vanilla OS. 
+date: 2023-10-26
+layout: article
+authors: 
+    - Phosphorus-M
+    - amsyarzero
+published: true
+---
+
+## Introduction
+
+[Vivaldi](https://vivaldi.com/) is a freeware, cross-platform web browser developed by Vivaldi Technologies, a company founded by Opera Software co-founder and former CEO Jon Stephenson von Tetzchner and Tatsuki Tomita. Vivaldi is based on Chromium, the open-source project behind Google Chrome. Vivaldi is intended to be a customizable, functional, and practical browser for the web.
+
```

Comment:
```suggestion
[Vivaldi](https://vivaldi.com) is a freeware, cross-platform web browser based on Chromium, the open-source project behind Google Chrome. Vivaldi is intended to be a customizable, functional, and practical browser for the web.

```

I think we can keep the description concise here.

Reply:
Should use ** to make the link bold.